browser/policy/BUILD.gn

```diff
@@ -58,3 +58,36 @@ source_set("brave_simple_policy_map") {
     deps += [ "//brave/components/ipfs" ]
   }
 }
+
+static_library("policy") {
```

Comment:
this should be under browser/policy/handlers to match upstream

Reply:
makes sense! moved handler code to browser/policy/handlers